src/app/coursegrab/dao/courses_dao.py

```diff
@@ -1,10 +1,23 @@
 from . import *
+from app.coursegrab.utils import scraper
 
 
 def get_course_by_catalog_num(catalog_num):
     return Course.query.get(catalog_num)
 
 
+def get_subject_by_catalog_num(catalog_num):
+    course = Course.query.filter(Course.catalog_num == catalog_num).first()
```

Comment:
`course = get_course_by_catalog_num(catalog_num)`

Reply:
oh word, this was before I made `catalog_num` the primary key, woops